subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/billing/PlayBillingManager.kt

```diff
@@ -75,6 +75,7 @@ interface PlayBillingManager {
         activity: Activity,
         planId: String,
         externalId: String,
+        offerId: String? = null,
```

Comment:
nit: I doesn't seem like we need that default value here. Right now, `SubscriptionManager` is always passing a value for `offerId`. Without that default value it will be harder to miss that parameter e.g., when refactoring something in the future.